src/Core/Services/StateService.cs

```diff
@@ -1551,28 +1480,6 @@ private async Task ScaffoldNewAccountAsync(Account account)
             {
                 state.Accounts = new Dictionary<string, Account>();
             }
-            if (state.Accounts.ContainsKey(account.Profile.UserId))
-            {
-                // Run cleanup pass on existing account before proceeding
-                await RemoveAccountAsync(account.Profile.UserId, false);
```

Comment:
I see that `RemoveAccountAsync(...)` does more stuff other than things related to global properties of this PR. Are you sure it's ok to remove this line?

Reply:
Yes, the other things were a pre-login safety measure to pave over any existing data in case a previous logout didn't execute cleanly for any reason, and that situation no longer applies.